packages/wpcom-checkout/src/transformations.ts

```diff
@@ -188,9 +226,17 @@ function getCreditsUsedByCart( responseCart: ResponseCart ): number {
 	return isFullCredits ? responseCart.sub_total_with_taxes_integer : responseCart.credits_integer;
 }
 
+function getYearlyVariantFromProduct( product: ResponseCartProduct ) {
+	return product.product_variants.find( ( variant ) => 12 === variant.bill_period_in_months );
+}
+
+function isBiYearlyProduct( product: ResponseCartProduct ) {
```

Comment:
Minor but "Biennial", meaning once every two years, might be a better term to use than "Biyearly" since the latter could mean "twice per year".

Also, does this do the same thing as [isBiennially](https://github.com/Automattic/wp-calypso/blob/77766fbd0769fedc38751ffa24f18470cb20328e/packages/calypso-products/src/is-biennially.ts#L3-L5)? If so, we could probably use that function here because `@automattic/calypso-products` is already a dependency of this package.

Reply:
Yes, indeed, that function works exactly the same, replacing my custom one with this one. Thanks!